hedera-node/src/main/java/com/hedera/services/state/migration/MigrationRecordsManager.java

```diff
@@ -224,13 +270,91 @@ private void publishContractFreeAutoRenewalRecords() {
                         });
     }
 
-    @VisibleForTesting
-    void setSideEffectsFactory(Supplier<SideEffectsTracker> sideEffectsFactory) {
-        this.sideEffectsFactory = sideEffectsFactory;
+    private void publishTraceabilityMigrationRecords() {
+        final var contractStorageMap = contractStorage.get();
+        final var allContractsStateChangesBuilder = ContractStateChanges.newBuilder();
+        accounts.get()
+                .forEach(
+                        (id, account) -> {
+                            if (account.isSmartContract() && !account.isDeleted()) {
```

Comment:
I'm not sure we want to skip deleted accounts, we should sync with @steven-sheehy

Reply:
I think we should include deleted accounts.

Reply:
Fixed.